tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/DevOpsService.cs

```diff
@@ -443,9 +448,12 @@ private async Task<List<WorkItem>> FetchWorkItemsAsync(string query)
             {
                 var workItemClient = connection.GetWorkItemClient();
                 var result = await workItemClient.QueryByWiqlAsync(new Wiql { Query = query });
-                if (result != null && result.WorkItems != null)
+                logger.LogInformation($"{result.ToString()}");
+                if (result != null && result.WorkItems != null && result.WorkItems.Any())
```

Comment:
Devops query returns empty result instead of null and I started running into this after deleting a work item previously created for same spec PR.